sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample9_DifferentiateOutputModelsTrainedWithAndWithoutLabels.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.AI.FormRecognizer.Training;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
+    {
+        /// This sample demonstrates the differences in output that arise when StartRecognizeCustomForms
+        /// is called with custom models trained with labels and without labels.
+
+        [Test]
+        public async Task OutputModelsTrainedWithLabels()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+            string trainingFileUrl = TestEnvironment.BlobContainerSasUrl;
+            string formFilePath = FormRecognizerTestEnvironment.CreatePath("Form_1.jpg");
+
+            FormTrainingClient trainingClient = new FormTrainingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            // Model trained with labels
+            CustomFormModel modelWithLabels = await trainingClient.StartTraining(new Uri(trainingFileUrl), useTrainingLabels: true).WaitForCompletionAsync();
+
+            using (FileStream stream = new FileStream(formFilePath, FileMode.Open))
+            {
+                RecognizedFormCollection formsWithLabels = await client.StartRecognizeCustomForms(modelWithLabels.ModelId, stream).WaitForCompletionAsync();
+
+                // With a form recognized by a model trained with labels, the 'field.Name' key will be its
+                // training-time label. Note that Label data is not returned for model trained with labels.
```

Comment:
Can we say: the label that you gave it at training time?  (Just to make it a little more clear)

Also: where we say "Label data not returned.." Could we explain why?  It would be good to use this opportunity to explain the difference between Form Field Labels ([common ](https://www.w3docs.com/learn-html/html-label-tag.html)in UI frameworks) and Training-Time Labels.  (Otherwise people might think we are crazy ;-))

Reply:
>explain the difference between Form Field Labels (common in UI frameworks) and Training-Time Labels

Ohh interesting. I wonder if this will be the right place for it though. I mean, I will hope for people reading the advance samples, that they already "looked" at the Readme and got somehow familiar with the service, to understand what labels in this service mean.

Reply:
I'd rather have the same information in more places to make sure people don't miss it.  At a minimum, let's link back to where this is discussed in the README.

Reply:
Added information a the beginning of the sample